test/tools/libtesteth/BlockChainHelper.cpp

```diff
@@ -222,7 +222,7 @@ void TestBlock::mine(TestBlockChain const& _bc)
         // mining!");
         cnote << "Mining block difficulty of:  " << block.info().difficulty();
         dev::eth::mine(block, blockchain, blockchain.sealEngine());
-        blockchain.sealEngine()->verify(JustSeal, block.info());
+        blockchain.sealEngine()->verify(CheckNothingNew, block.info());
```

Comment:
What this change does?

Reply:
I told you to change timestamp in blockheader. nobody agrees. now you ask me why and nobody unserstands. timestamp in blockheader has value of signed int. and it is incorrect by default when you create the blockheader. 

The intention of this check was to validate that that blocheader is constructed correct (but without signature). however if it throws then mining does not work. I guess this line could be removed, but who knows what might happen. I could try to remove line 225 but that will require to check that BC test filling still work.

Reply:
@chfast This was my suggestion to remove `JustSeal`, because it is not being handled correctly, and it's used only in json tests, but for them using `CheckNothingNew` is also fine